content/dart/concepts/queue/terms/clear/clear.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.clear()'
+Description: 'A method that removes all elements in a queue.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Dart'
+  - 'Classes'
+  - 'Queues'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+In [`Dart`](<[url](https://www.codecademy.com/resources/docs/dart)>), the **`.clear()`** method in [`Queue`](<[url](https://www.codecademy.com/resources/docs/dart/queue)>) removes all elements in the queue. The size of the queue becomes zero.
```

Comment:
```suggestion

In Dart, the **`.clear()`** method in Queue removes all elements in the queue. The size of the queue becomes zero.
```
It doesn't make sense to add those links as it is clear that this entry is under dart topic and queue concept.